src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -1722,10 +1722,13 @@ private void DoRunspaceInitialization(bool importSystemModules, bool skipProfile
                 // configuration to get the shell ID from, so we'll use the default...
                 string shellId = null;
                 if (_configuration != null)
-                    shellId = _configuration.ShellId;
+                        shellId = _configuration.ShellId;
```

Comment:
I don't see a tab when I look at the file.  May be an issue with GitHub diff.

---
In reply to: [75389938](https://github.com/PowerShell/PowerShell/pull/1845#discussion_r75389938) [](ancestors = 75389938)